router/image/templates/nginx.conf

```diff
@@ -157,6 +158,12 @@ http {
 
             proxy_next_upstream         error timeout http_502 http_503 http_504;
 
+            {{ if ne $enforceHttps "false" }}
```

Comment:
IMO it's more clear to be explicit about true rather than false, so either checking if the key exists (`{{ if $enforceHttps }}`) or if the key is set explicitly to "true" (`{{ if eq $enforceHttps "true" }}`) is preferred.

We should've used that for the other key values, but that's a behavioural change so we can't change it now.